plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4516,14 +4474,14 @@ class AzureNetworkWatcher(AzureResource):
 
     def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
         locations_and_ids_in_vn = self.fetch_resources(
-            builder=builder,
+            builder,
             service="network",
             api_version="2023-05-01",
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualNetworks",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=lambda r: r["location"],
-            binding_property=lambda r: r["id"],
+            first_property=lambda r: r["location"],
+            second_property=lambda r: r["id"],
         )
```

Comment:
This call is not required - it fetches all virtual networks, which have been already fetched and are available in the graph.